src/coreclr/src/vm/method.cpp

```diff
@@ -5429,6 +5429,14 @@ BOOL MethodDesc::HasUnmanagedCallersOnlyAttribute()
         WellKnownAttribute::UnmanagedCallersOnly,
         nullptr,
         nullptr);
+    if (hr != S_OK)
+    {
+        hr = GetCustomAttribute(
+            WellKnownAttribute::NativeCallableInternal,
```

Comment:
Can we open issue to remove it and link it from here?

Reply:
Done https://github.com/dotnet/runtime/issues/37622.